### DISCUSSION TITLE
Multiple labels wrapper and fixed width

### DISCUSSION THREAD

**Discussion by PedroJBrito** [chartjs/chartjs-plugin-datalabels#363] [Q&A]
Hi guys,

I need to implement a design that requires multiple labels, like follow

![image](https://user-images.githubusercontent.com/94718876/211218214-59031c20-a11f-41a6-9d27-9049327d0b3e.png)

however the required design needs something like a wrapper around the multiple labels that I don't know if it is possible to achieve. Besides that I also need a fixed width for the whole content and to slice the description according to overflows that width. Is it possible to achieve something like this? Can you give me some tips?

Thank you for your time.
Kind regards

**Comment by Udemology** [comment]
Is this a horizontal bar with borderradius?  
And the #8 = value and the '5 Health Care...' would be the label text?

**Reply by PedroJBrito** [reply]
Hi,

This is a point in a radar chart. The 8 value and the '5 Health...' would be the label, however the 8 is supposed to have a different style.

Thank you once again.

**Reply by Udemology** [reply]
Do you have a zoomed out version of this chart? Because I struggle what you want to do. A bar chart in radar chart or some combo chart type.

**Reply by PedroJBrito** [reply]
Hi,

yeah, sure, this is what I want to achieve:

![image](https://user-images.githubusercontent.com/94718876/212300670-bf5df7f4-987c-4060-b29b-77bc30b33141.png)

Thank you for your time

**Reply by Udemology** [reply]
Alright, this has some complicated parts but manage to do it without the plugin. If you used the plugin I am sure it could simplify the value in the aqua green center. So you can see if it works. I created it from scratch building it based on the jitter radar chart: https://youtu.be/Zem2JDZ3Hv0

**Reply by PedroJBrito** [reply]
Thanks a lot for your help, I really appreciate it 👍
